README.md

```diff
@@ -1,9 +1,6 @@
 # TDLight Telegram Bot API
 The official documentation is [here](https://tdlight-team.github.io/tdlight-telegram-bot-api/).
 
-TDLight Telegram Bot API is a fork of the official Telegram Bot API, focused on memory footprint and performance.
-
-TDLight Telegram Bot API is 100% compatible with the official version.
 
```

Comment:
Keep this lines, maybe replace them with:

TDLight Telegram Bot API is a fork of the official Telegram Bot API.

TDLight Telegram Bot API is 100% compatible with the official version.